icepyx/tests/test_granules.py

```diff
@@ -592,16 +592,14 @@ def test_no_granules_in_search_results():
 
 def test_correct_granule_list_returned():
     reg_a = ipx.Query(
-        "ATL06", [-55, 68, -48, 71], ["2019-02-20", "2019-02-28"], version="2"
+        "ATL06", [-55, 68, -48, 71], ["2019-02-20", "2019-02-28"]
```

Comment:
If we don't specify a version here, this test will fail as soon as a new version of the datasets is available. Updating to v3 is a good idea though.

Reply:
good point.  I added `version="3"`

Reply:
I'm thinking of a query

`reg_a = ipx.query(granule=list_of_granules)`

The product name, dates, version etc could be parsed from the granule file names.

I have a workflow to do this with the CMR.  All thoughts on this welcome.